compiler/ballerina-lang/src/main/java/io/ballerina/compiler/api/symbols/TupleMemberTypeSymbol.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright (c) 2022, WSO2 LLC. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.ballerina.compiler.api.symbols;
+
+/**
+ * Represents a tuple member type symbol.
+ *
+ * @since 2201.4.0
+ */
+public interface TupleMemberTypeSymbol extends Symbol, Annotatable {
```

Comment:
```suggestion
 * Represents a tuple member symbol.
 *
 * @since 2201.4.0
 */
public interface TupleMemberSymbol extends Symbol, Annotatable {
```

Type is just the type?

Reply:
+1. Please check my thoughts on https://github.com/ushirask/ballerina-lang/pull/54#discussion_r1062132733